helper/helper.go

```diff
@@ -0,0 +1,23 @@
+package helper
+
+import (
+	"context"
+	"time"
+
+	"github.com/ONSdigital/dp-frontend-homepage-controller/config"
+	"github.com/ONSdigital/log.go/v2/log"
+)
+//checks if current time is equal/after Census First Results time
```

Comment:
Comment on exported function/variable should always start with the function/variable name, like so:

```suggestion
// CheckTime checks if the current time is equal/before the value of time variable passed into function
```

I've reworded the comment too, as we can generalise this function so it is not specific to CensusFirstResults